package.json

```diff
@@ -31,7 +31,16 @@
 		"./2015/": "./2015/",
 		"./helpers/": "./helpers/",
 		"./5/": "./5/",
-		"./": "./"
+		"./": "./",
+		"./2020/*": "./2020/*.js",
+		"./2019/*": "./2019/*.js",
+		"./2018/*": "./2018/*.js",
+		"./2017/*": "./2017/*.js",
+		"./2016/*": "./2016/*.js",
+		"./2015/*": "./2015/*.js",
+		"./helpers/*": "./helpers/*.js",
+		"./5/*": "./5/*.js",
+		"./*": "./*.js"
```

Comment:
unfortunately, this causes ` Cannot find module '$PWD/node_modules/es-abstract/2020/RequireObjectCoercible.js.js'` or similar in node 16.6, 12.23, and 12.17 when I try to import with the extension - when i try to import without the extension, I get `Cannot find module '$PWD/node_modules/es-abstract/2020/RequireObjectCoercible'` in node v12.17 - so this doesn't suffice.

If you remove the `.js` from all the RHS's of these lines, then ESM works, but CJS (the vastly more important consumer for the foreseeable future) is no longer able to require them without extensions.